pyaerocom/colocation/colocation_utils.py

```diff
@@ -976,7 +976,7 @@ def colocate_gridded_ungridded(
         "from_files": files,
         "from_files_ref": None,
         "colocate_time": colocate_time,
-        "obs_is_clim": True if isinstance(use_climatology_ref, ClimatologyConfig) else False,
+        "obs_is_clim": (True if isinstance(use_climatology_ref, ClimatologyConfig) else False),
```

Comment:
`isinstance` returns a boolean, the `True if X else False` construct is redundant. You could write it like this

``` python
        "obs_is_clim": isinstance(use_climatology_ref, ClimatologyConfig),
```

Reply:
this is not my code,,,  I had to merge main-dev